compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -828,6 +828,28 @@ class PowTestGraph final : public SimpleTestGraph
   luci::CircleConst *_const = nullptr;
 };
 
+class ResizeBilinearTestGraph final : public luci::test::TestIOGraph
+{
+public:
+  void init(void)
+  {
+    TestIOGraph::init({1, 4, 4, 1}, {1, 8, 8, 1});
+
+    _size = create_const<Type::S32, int32_t>(g(), {2}, {8, 8});
+    _resize_bilinear = g()->nodes()->create<luci::CircleResizeBilinear>();
+    {
+      _resize_bilinear->input(input());
+      _resize_bilinear->size(_size);
+    }
+    output()->from(_resize_bilinear);
+
+    set_minmax_to_non_const(g(), -1, 1);
+  }
+
+  luci::CircleResizeBilinear *_resize_bilinear = nullptr;
```

Comment:
it would be better to add scope `private:` above this line. Other `XXXTestGraph` classes uses `public:` but I think it would be better to change them also.

Reply:
@seanshpark Sure :-D

Reply:
> but I think it would be better to change them also(in another PR)

I will make a change in a separate PR. :-D

Reply:
There is no `private:` scope here. Please add in this PR for this file.

Reply:
Sure. updated